.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           path: 'play/target/*/*/*.jar'
           retention-days: 1
   publish:
-    if: github.event_name != 'pull_request'
+    if: ${{ (github.event_name != 'pull_request') && (github.repository == 'lucidsoftware/opentracing-playframework') }}
```

Comment:
without this, would it try to publish from forks? Or how come we need this check now?

Reply:
it would try to publish on forks, but it would fail because they wouldn't have access to the sonatype credentials.